examples/glusterfs/service/glusterfs-service.json

```diff
@@ -0,0 +1,19 @@
+{
+    "apiVersion": "v1beta3",
+    "id": "glusterfs",
+    "kind": "Endpoints",
+    "metadata": {
+        "name": "glusterfs-cluster"
+    },
+    "Endpoints": [
+        {
+            "ip": "10.16.154.81"
```

Comment:
Optional:

Is there an easy way to generate this file for the user.  Maybe a small shell script or go program that does `gluster peer status` and then resolves the peer hostnames and generates the Endpoints list.